docs/advanced/storagenetwork.md

```diff
@@ -290,6 +290,97 @@ metadata:
 Omitted...
 ```
 
+#### Step 4
+
+The storage network is dedicated to [internal communication between Longhorn pods](#same-physical-interfaces), resulting in high performance and reliability. However, the storage network still relies on the [external network infrastructure](../networking/deep-dive.md#external-networking) for connectivity (similar to how the [VM VLAN network](../networking/harvester-network.md#create-a-vm-with-vlan-network) functions). When the external network is not connected and configured correctly, you may encounter the following issues:
+
+    - The newly created VM becomes stuck at the `Not-Ready` state.
+
+    - The `longhorn-manager` pod logs include error messages.
+
+    Example:
```

Comment:
The rendering still has issues. Might need to fix the indent.
![image](https://github.com/user-attachments/assets/46ef69dd-6320-497f-b69a-372e9d72da96)

Reply:
The updated doc will look like:
![image](https://github.com/user-attachments/assets/bf838493-199f-4c98-ab62-20b07ba822fd)